internal/internal_flags.go

```diff
@@ -37,7 +37,10 @@ const (
 	// LimitChangeVersionSASize will limit the search attribute size of TemporalChangeVersion to 2048 when
 	// calling GetVersion. If the limit is exceeded the search attribute is not updated.
 	SDKFlagLimitChangeVersionSASize = 1
-	SDKFlagUnknown                  = math.MaxUint32
+	// LimitChangeVersionSASize return errors to child workflow execution future if the child workflow would
```

Comment:
Wrong flag name in the comment here? (Might be a GH rendering glitch on my end)

Reply:
no good catch, just a typo